src/acquisition/covidcast/csv_importer.py

```diff
@@ -52,18 +52,21 @@ class CsvRowValue:
 class CsvImporter:
   """Finds and parses covidcast CSV files."""
 
+  # set of allowed resolutions (aka "geo_type")
+  GEOGRAPHIC_RESOLUTIONS = {'county', 'hrr', 'msa', 'dma', 'state', 'hhs', 'nation', "hsa", "hsa_nci"}
```

Comment:
I think we should keep `hsa` out of the list, at least for now -- if/when we truly need it, itll be easy enough to put back.  This is partly because the other (Dartmouth) definition of HSA will be incompatible with this one and we dont want ambiguity about which is being used.  Keep the length sorting and associated comment though, in case people forget about that case in the future!
```suggestion
  GEOGRAPHIC_RESOLUTIONS = {'county', 'hrr', 'msa', 'dma', 'state', 'hhs', 'nation', 'hsa_nci'}
```